docs/operations/java.md

```diff
@@ -0,0 +1,93 @@
+---
+id: java
+title: "Java runtime"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+Apache Druid is written in Java and requires a Java runtime. This page provides details about obtaining and configuring
+a Java runtime for Druid.
+
+## Selecting a Java runtime
+
+Druid fully supports Java 8 and 11. The project team recommends Java 11.
+
+The project team does not recommend running with Java 17, because certain Druid functionality is not currently
+compatible with Java 17.
+
+There are many free and actively-supported Java runtime environments available, such as
+[Corretto](https://aws.amazon.com/corretto/) and [Zulu](https://www.azul.com/downloads/?package=jdk#download-openjdk).
```

Comment:
should we also mention Adoptium Temurin https://adoptium.net/temurin/releases/ here as one of the non-vendor-specific alternatives?

Reply:
Good idea. I wasn't aware of that one. Added it.